channel2/message.go

```diff
@@ -41,6 +41,7 @@ const (
 	ReplyForHeader                  = 1
 	ResultSuccessHeader             = 2
 	HelloRouterAdvertisementsHeader = 3
+	HelloVersion                    = 4
```

Comment:
Should probably be `HelloVersionHeader` to align with the other headers above.

Reply:
Ahh yes. Consider it done.